modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties

```diff
@@ -1,3 +1,4 @@
+a-rule-is-applied-to-this-field=A rule is applied to this field. By deleting it, it will also break the rule that is using it.
```

Comment:
@alinedoleron, please separate this commit on 2 commits, one just adding the missing language keys and another one with the result of the `buildLang` command.